client/views/hooks/useUpgradeTabParams.ts

```diff
@@ -0,0 +1,16 @@
+import { useCallback } from 'react';
+import { useQuery } from 'react-query';
+
+import { UpgradeTabVariants } from '../../../lib/getUpgradeTabType';
+import { useEndpoint } from '../../contexts/ServerContext';
+
+export const useUpgradeTabParams = (): [data: UpgradeTabVariants | false, trialEndDate: string | undefined, isLoading: boolean] => {
```

Comment:
Why not return `UseQueryResult<{ tabType: UpgradeTabVariants; trialEndDate: string }, Error>` and let the caller use the query as it wants?

Reply:
Fixed :)